backend/pkg/httpserver/list_missing_one_implementation_counts_test.go

```diff
@@ -309,6 +324,41 @@ func TestListMissingOneImplementationCounts(t *testing.T) {
 					"browser=edge&browser=firefox&browser=safari&"+
 					"startAt=2000-01-01&endAt=2000-01-10&page_token"+*badPageToken, nil),
 		},
+		{
+			name: "400 case - no matching mobile browser",
+			mockConfig: &MockListMissingOneImplCountsConfig{
+				expectedTargetBrowsers: []string{"chrome"},
+				expectedOtherBrowsers: [][]string{
+					{"edge"},
+					{"firefox"},
+					{"safari"},
+				},
+				expectedStartAt:   time.Date(2000, time.January, 1, 0, 0, 0, 0, time.UTC),
+				expectedEndAt:     time.Date(2000, time.January, 10, 0, 0, 0, 0, time.UTC),
+				expectedPageSize:  100,
+				expectedPageToken: badPageToken,
+				page:              nil,
+				pageToken:         nil,
+				err:               backendtypes.ErrInvalidPageToken,
+			},
+			expectedGetCalls: []*ExpectedGetCall{
+				{
+					Key: `ListMissingOneImplementationCounts-{"browser":"edge","Params":{"startAt":"2000-01-01",` +
+						`"endAt":"2000-01-10","include_baseline_mobile_browsers":true,` +
+						`"browser":["chrome","firefox","safari"]}}`,
+					Value: nil,
+					Err:   ErrNoMatchingMobileBrowser,
```

Comment:
This is the error returned from the cache layer.
```suggestion
					Err:   cachetypes.ErrCachedDataNotFound,
```